react-search-component/my-app/src/SearchBar.tsx

```diff
@@ -0,0 +1,7 @@
+type InputProps = {
+  handleChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
Since it's an input prop, it should start with `on`.

I also find it more convenient to _not_ use the React.XxxEvent types when declaring functions in my props. It usually works much better (and gives better TS checking) if you explicitly declare the function type you expect. In this case, simply `() => void` should work. (Though that may require you to change your `onChange` implementation below -- a very small price to pay.)